src/migrate/azext_migrate/_help.py

```diff
@@ -0,0 +1,276 @@
+# coding=utf-8
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+# pylint: disable=too-many-lines
+# pylint: disable=line-too-long
+from knack.help_files import helps  # pylint: disable=unused-import
+
+
+helps['migrate location'] = """
+    type: group
+    short-summary: migrate location
+"""
+
+helps['migrate location check-name-availability'] = """
+    type: command
+    short-summary: Checks whether the project name is available in the specified region.
+    examples:
+      - name: LocationCheckNameAvailability_AlreadyExists
+        text: |-
+               az migrate location check-name-availability --location-name "eastus" --name \\
+               "existingprojectname"
+      - name: LocationCheckNameAvailability_Available
+        text: |-
+               az migrate location check-name-availability --location-name "eastus" --name \\
+               "newprojectname"
+"""
+
+helps['migrate assessment-options'] = """
+    type: group
+    short-summary: migrate assessment-options
+"""
+
+helps['migrate assessment-options show'] = """
+    type: command
+    short-summary: Get the available options for the properties of an assessment.
+    examples:
+      - name: AssessmentOptions_Get
+        text: |-
+               az migrate assessment-options show --location-name "SoutheastAsia"

```

Comment:
is the location not the global location?

Reply:
Yes. it is. This should have been changed into location but I am wondering how about parameters like location-id. Should it also be changed into location?